assignments/arrays.js

```diff
@@ -63,24 +63,34 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year , inventory[32].car_make , inventory[32].car_year}` );
```

Comment:
This solution works for now but what happens if someone deletes the first few items in the array. How would you then solve it?